Content.Server/Wires/BaseWireAction.cs

```diff
@@ -1,12 +1,16 @@
 using Content.Server.Power.Components;
 using Content.Server.Power.EntitySystems;
+using Content.Shared.Administration.Logs;
+using Content.Shared.Database;
 using Content.Shared.Wires;
 
 namespace Content.Server.Wires;
 
 /// <summary><see cref="IWireAction" /></summary>
 public abstract class BaseWireAction : IWireAction
 {
+    [Dependency] private ISharedAdminLogManager _adminLogger = default!;
+
```

Comment:
[Dependency] just means when IoCManager.InjectDependencies is run it will get injected so it's unnecessary if you're resolving elsewhere.